app/models/participant_declaration.rb

```diff
@@ -64,12 +64,17 @@ class ParticipantDeclaration < ApplicationRecord
   scope :paid_npqs_for_lead_provider, ->(lead_provider) { paid_for_lead_provider(lead_provider).npq }
   scope :paid_uplift_for_lead_provider, ->(lead_provider) { paid_for_lead_provider(lead_provider).uplift }
 
+  # TODO: Voiding paid should trigger clawbacks, but currently OOS
```

Comment:
Let's wait for this https://ukgovernmentdfe.slack.com/archives/C01BE7G56F7/p1637577636468600

and remove this comment. 

I do really think,  that `TODO` should not live in the code, as the PMs have no visibility on it hence ignore this is a thing and can't really factor this into any sprint, leaving the code with lots of potentially never addressed TODOs.

Reply:
@StupidCodeFactory We'll already have a ticket for this — it's more an FYI to ourselves that we have not yet solved this problem :)